Symfony/CS/Fixer/PSR2/BracesFixer.php

```diff
@@ -367,7 +369,10 @@ private function fixSpaceAroundToken(Tokens $tokens)
         for ($index = $tokens->count() - 1; 0 <= $index; --$index) {
             $token = $tokens[$index];
 
-            if ($token->isGivenKind($controlTokens) || $token->isGivenKind(T_USE)) {
+            // Declare tokens don't follow the same rules are other control statements
```

Comment:
It's listed in the `getControlTokens` method - https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.11/Symfony/CS/Fixer/PSR2/BracesFixer.php#L529 -  and it's under the control structures section of the PHP docs -  
http://php.net/manual/en/control-structures.declare.php